libs/src/services/ethContracts/serviceTypeManagerClient.js

```diff
@@ -2,12 +2,25 @@ const Utils = require('../../utils')
 const GovernedContractClient = require('../contracts/GovernedContractClient')
 
 class ServiceTypeManagerClient extends GovernedContractClient {
-  async setServiceVersion (serviceType, serviceVersion, privateKey = null) {
+  /**
+   *
+   * @param {String} serviceType Type of service to set the version, either `discovery-node` or `content-node`
+   * @param {String} serviceVersion Version string to set on chain
+   * @param {String?} privateKey Optional privateKey to pass along to web3Manager sendTransaction
+   * @param {String?} dryRun Optional parameter to return the generated parameters without sending tx
```

Comment:
I know this is piped via a cli arg so that's why it's a string, but might be a little error prone because we just check for it's existence here, and in the mainnet.js it checks for the string - so it there was a typo in false ie "fals" it would set it. 
Maybe from the cli make it an optional flag instead of positional arg? 
Honestly this is probs fine, just wanted to point it out.

Reply:
it's supposed to be boolean, i modified the script to parse out a boolean value from the positional arg but i also set a boolean default of true. i change the docstring